tests/unit/test_tasks.py

```diff
@@ -51,7 +51,7 @@ def header(request, thing):
 
     def test_call(self, monkeypatch):
         request = pretend.stub()
-        registry = pretend.stub()
+        registry = pretend.stub(settings={"warehouse.ip_salt": "peppa"})
```

Comment:
I use `pepa` from Salt-N-Pepa 🧂 🌶️  - this is the spelling for Peppa Pig 🐷 . Don't change a thing! 🤣